Mathlib/Algebra/Group/AddChar.lean

```diff
@@ -74,7 +74,7 @@ namespace AddChar
 section Basic
 -- results which don't require commutativity or inverses
 
-variable {A M : Type*} [AddMonoid A] [Monoid M]
+variable {A B M N : Type*} [AddMonoid A] [AddMonoid B] [Monoid M] [Monoid N] {ψ : AddChar A M}
```

Comment:
Seems a bit strange to declare `{ψ : AddChar A M}` as standing notation when it is only used twice much further down (most lemmas take the explicit argument)

Reply:
I think that's okay. Furthermore since [I don't have cache](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/cache.20misses.20aesop) I won't adventure myself at removing `{ψ : AddChar A M}` to put it only where needed.